etc/inc/config.console.inc

```diff
@@ -202,7 +204,8 @@ hitting 'a' to initiate auto detection.
 EOD;
 
 		do {
-			echo "\n" . gettext("Enter the WAN interface name or 'a' for auto-detection:") . " ";
+			echo "\n" . gettext("Enter the WAN interface name or 'a' for auto-detection") . " ";
+			echo "\n" . gettext("(") . $ifsmallist . gettext(" or a): ");
```

Comment:
In cases like this, you are better off using a single gettext with printf(), like this:
```
printf(gettext("Your message with %s parameter"), $var);
```
This way who is translating the message can modify it according the language structure, and the message makes sense